lua/ulx/modules/sh/yeet.lua

```diff
@@ -0,0 +1,37 @@
+local CATEGORY_NAME = "Fun"
+
+local MAX_STRENGTH = 500000	
+local DEFAULT_STRENGTH = 6000 
+
+function yeetPlayer( caller, targets, strength )
+	
+    for _, v in pairs( targets ) do
+		
+    if not v:IsPlayer() then return end
+    if not v:Alive() then return end
```

Comment:
you wouldnt want to return here because if you have multiple players and the first player isnt alive then Nobody will get yeeted

I would suggest moving all the logic for yeeting one player into its own function then you could do something like
```lua
if v:IsAlive() then yeetPlayer( v ) end
```